subsys/bluetooth/host/gatt.c

```diff
@@ -1105,8 +1105,8 @@ void bt_gatt_foreach_attr_type(u16_t start_handle, u16_t end_handle,
 			for (i = 0; i < static_svc->attr_count; i++, handle++) {
 				struct bt_gatt_attr attr;
 
-				memcpy(&attr, &static_svc->attrs[i],
-				       sizeof(attr));
+				(void)memcpy(&attr, &static_svc->attrs[i],
+						sizeof(attr));
```

Comment:
Same here. Seems like the script only uses tabs, rather than tabs + spaces to accurately align with the opening parenthesis of the first line? There are many more places for Bluetooth code I looked at, but it's the same thing so I won't bother pointing it out for each case.

Reply:
@jhedberg It is everywhere like that ;), it is indeed all aligned with tabs

Reply:
@jhedberg I'm confuse now. checkpatch.pl should catch code styles issues  right ?
Just for sake of curiosity I also tried uncrustify using Zephyr's configuration, and uncrustify complained about the mentioned alignment. So far so good, the problem is that is also spot *a lot* of other issues in this file. e.g:

```
typedef enum __packed {
                ATT_COMMAND,
...
u8_t       expect_len;
... 
{ BT_ATT_OP_MTU_REQ,
        sizeof(struct bt_att_exchange_mtu_req),
...
```
So, how are we enforcing the code style ?

Reply:
AFAIK, checkpatch allows both aligning only with tabs, and adding a few spaces after the tabs (to align to the bracket). It just does not let you start with spaces.